src/utils/swap.utils.ts

```diff
@@ -205,8 +208,8 @@ export const getSwapTransferParams = async (
     approvesParams = approveInputToken;
     revokesParams = revokeInputToken;
     swapParams = [];
-    const xtzSwapOut = tzToMutez(chains.xtzTree.tokenOutAmount, THREE_ROUTE_XTZ_TOKEN.decimals);
-    const tzbtcSwapOut = tzToMutez(chains.tzbtcTree.tokenOutAmount, THREE_ROUTE_TZBTC_TOKEN.decimals);
+    const xtzSwapOut = tzToMutez(new BigNumber(chains.xtzTree.tokenOutAmount), THREE_ROUTE_XTZ_TOKEN.decimals);
+    const tzbtcSwapOut = tzToMutez(new BigNumber(chains.tzbtcTree.tokenOutAmount), THREE_ROUTE_TZBTC_TOKEN.decimals);
```

Comment:
mb the fallback (`?? ZERO`) is needed here? did you check that swap works fine if those values appear to be `NaN`?

Reply:
No, I didn't check. Actually, I have never seen the case when `tokenOutAmount` is undefined